x-pack/plugins/data_usage/server/services/autoops_api.ts

```diff
@@ -145,22 +146,22 @@ export class AutoOpsAPIService {
               },
             }
           );
-          throw new AutoOpsError(withRequestIdMessage(AGENT_CREATION_FAILED_ERROR));
+          throw new AutoOpsError(withRequestIdMessage(AUTO_OPS_FAILED_ERROR));
```

Comment:
Use updated variable name.

```suggestion
          throw new AutoOpsError(withRequestIdMessage(AUTO_OPS_REQUEST_FAILED_PREFIX));
```